### PR TITLE
Core validators

### PR DESCRIPTION
**Pull request by nemvalid**

### CODE REVIEW

File: public/views/partials/lib/validations/uniqueness.liquid [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      assign scope_name = ''
+    endif
+
+    if exclude_name
+      assign exclude_value = object[exclude_name]
+    else
+      assign exclude_name = ''
+    endif
+
+    graphql r = 'records/count', property_name: field_name, property_value: value, not_id: object_id, table: table, scope_name: scope_name, scope_value: scope_value, exclude_name: exclude_name, exclude_value: exclude_value
```
Comment (piotrze) [inline public/views/partials/lib/validations/uniqueness.liquid:19]: @nemvalid This query is missing.

**Comment by piotrze** [comment]
@nemvalid Please add tests. My guess is that `uniqueness` validation is not working.

File: public/views/partials/lib/validations/hcaptcha.liquid [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{% comment %}
+  params: @params_with_hcaptcha
+          @c
+{% endcomment %}
+
+{% liquid
+  if context.constants.VERIFY_HCAPTCHA == "true" and hcaptcha_params
```
Comment (nemvalid) [inline public/views/partials/lib/validations/hcaptcha.liquid:7]: `if context.constants.VERIFY_HCAPTCHA == "true"` should be removed. looks like a poorly implemented feature flag, doesn't make sense in this context.

**Comment by hosszukalman** [comment]
We will need to add default translations

**Comment by hosszukalman** [comment]
> @nemvalid Please add tests. My guess is that `uniqueness` validation is not working.

@piotrze We don't have test module now but I totally agree, we will need to create a test framework for unit tests. We have a card about it here: https://trello.com/c/gFXXjM0i/109-unit-testing and I hope we will have time to do it in the next sprint

**Review by hosszukalman — APPROVED** [review]

**Pull request merged**